db/model/collector.js

```diff
@@ -257,10 +256,11 @@ module.exports = function collector(seq, dataTypes) {
   Collector.prototype.isAlive = function () {
     if (!this.lastHeartbeat) return false;
     const tolerance = collectorConfig.heartbeatLatencyToleranceMillis;
+    const interval = collectorConfig.heartbeatIntervalMillis;
     const now = Date.now();
     const lastHeartbeat = this.lastHeartbeat.getTime();
     const elapsed = now - lastHeartbeat;
-    return elapsed < tolerance;
+    return elapsed < interval + tolerance;
```

Comment:
I always find it easier/faster to read a line like this with totally unnecessary parentheses added, e.g.

```
return elapsed < (interval + tolerance);
```